src/adagrams.js

```diff
@@ -1,15 +1,127 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
+/* eslint-disable quotes */
```

Comment:
Totally ok in this context to disable a couple eslint rules but it's a slippery slope and in practice this would get noted in a code review. In practice, there will be an eslint configuration file that stipulates which rules should be in play and you'll have to abide by it. It's also a good practice to get in the habit of conforming to the code style espoused by them!